client/packages/requisitions/src/RnRForms/DetailView/DetailView.tsx

```diff
@@ -15,15 +15,15 @@ import { ActivityLogList } from '@openmsupply-client/system';
 import { Footer } from './Footer';
 import { AppBarButtons } from './AppBarButtons';
 import { ContentArea } from './ContentArea';
-import { RnRFormQuery, useRnRForm, useRnRFormContext } from '../api';
+import { oneTime, RnRFormQuery, useRnRForm, useRnRFormContext } from '../api';
 import { SidePanel } from './SidePanel';
 import { AutoSave } from './AutoSave';
 
 export const RnRFormDetailView = () => {
   const { id = '' } = useParams();
   const [isInitialising, setIsInitialising] = useState(true);
 
-  const setInitial = useRnRFormContext(state => state.setInitial);
+  const setInitial = useRnRFormContext(oneTime(state => state.setInitial));
```

Comment:
hmm, so i noticed that content area was refreshing due to some changes i've added, so added oneTime just in case